pastas/model.py

```diff
@@ -685,7 +685,14 @@ def initialize(
         manually. See the solve-method for a description of the arguments.
         """
         if noise is None and self.noisemodel:
-            noise = True
+            msg = (
+                "The default of noise=True is deprecated and will be changed to "
+                "False in a future version of pastas. Pass noise=True to retain "
+                "current behavior or noise=False to adopt the future default and "
+                "silence this warning."
```

Comment:
Should we add a reference to issue here: https://github.com/pastas/pastas/issues/667 ? Then I think users are informed and we can merge this..